team/bundles/org.eclipse.core.net/src/org/eclipse/core/net/proxy/linux/ProxyProviderLinux.java

```diff
@@ -11,7 +11,7 @@
  * Contributors:
  *     Red Hat Inc. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.core.net;
+package org.eclipse.core.net.proxy.linux;
```

Comment:
What do you think about making this internal package? I don't believe this is smth that should be exposed as API.

Reply:
There packages where not exported, but it can definitely be made more explicit.
Implemented it.